src/_includes/layouts/base.ejs

```diff
@@ -71,20 +73,14 @@
       <div class="hidden md:flex md:flex-shrink-0">
         <div class="flex flex-col w-64">
           <div class="flex flex-col flex-grow pt-5 pb-4 overflow-y-auto border-r border-gray-200">
-            <div class="flex items-center flex-shrink-0 px-4">
-              <%- include('../logo.ejs') %>
-            </div>
-            <div class="flex flex-col flex-grow mt-5">
-              <%- include('../menu.ejs') %>
-            </div>
+            <div class="flex items-center flex-shrink-0 px-4"><%- include('../logo.ejs') %></div>
+            <div class="flex flex-col flex-grow mt-5"><%- include('../menu.ejs') %></div>
```

Comment:
Zijn dit auto lint fixes? of jij zelf gedaan?  Ik vind het zo minder lekker lezen namelijk.

Reply:
> Zijn dit auto lint fixes? of jij zelf gedaan? Ik vind het zo minder lekker lezen namelijk.

Ja Prettier, wat dit project gebruikt. Dan moet je de prettierrc anders instellen, bijv. een printWidth van 80.

Reply:
Maar ja eens, dat eens dat het wat minder lekker lezen is, maar vindt dat vooral komen door de Tailwind classes persoonlijk :).